api/src/main/java/marquez/service/OpenLineageService.java

```diff
@@ -142,7 +142,16 @@ public CompletableFuture<Void> createAsync(LineageEvent event) {
             .thenAccept(
                 (update) -> {
                   if (event.getEventType() != null) {
-                    if (event.getEventType().equalsIgnoreCase("COMPLETE")) {
+                    boolean isStreaming =
+                        Optional.ofNullable(event.getJob())
```

Comment:
We can use the following method as described [here](https://github.com/MarquezProject/marquez/pull/2682/files#r1424584879):

```
event.getJob().isStreaming()
```